website/src/components/DashboardHeader/DashboardHeader.test.jsx

```diff
@@ -5,18 +5,27 @@ import {
 } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import { MemoryRouter } from 'react-router-dom';
+import firebase from 'firebase';
 import DashboardHeader from './DashboardHeader';
 
 configure({ adapter: new Adapter() });
 
-describe('DashboardHeader', () => { // eslint-disable-line no-undef
-  it('Expect DashboardHeader to match snapshot', () => { // eslint-disable-line no-undef
+describe('DashboardHeader', () => {
+  it('Expect DashboardHeader to match snapshot', () => {
+    jest.spyOn(firebase, 'auth')
+      .mockImplementation(() => ({
+        currentUser: {
+          displayName: 'testDisplayName',
+          email: 'test@test.com',
+          emailVerified: true,
```

Comment:
`emailVerified` should be false always in our case, because we're not sending email verification emails. Also, this field is never used so why have it in the mock?

Reply:
This is sample data  and would not affect or hit firebase it is just what our mock function for firebase returns.  I can modify it to set that field to false.

Reply:
ye but why have a field in mock when it's not used in real application

Reply:
True removed